src/FileWatcher/index.js

```diff
@@ -52,9 +53,7 @@ class FileWatcher extends EventEmitter {
         );
       });
       return watchers;
-    });
-
-    this._loading.done();
+    }, console.error.bind(console));
```

Comment:
let's remove this. I don't want node-haste to log to the console at all. All errors should be able to be caught and processed by clients of node-haste.